x-pack/plugins/infra/public/apps/common_providers.tsx

```diff
@@ -70,11 +73,15 @@ export const CoreProviders: React.FC<CoreProvidersProps> = ({
     pluginStart
   );
 
+  const MyEnvContextForPluginProvider = useKibanaEnvironmentContextProvider(plugins);
```

Comment:
suggestion: completely up to you
```suggestion
  const KibanaEnvContextForPluginProvider = useKibanaEnvironmentContextProvider(plugins);
```

Reply:
Done, thanks ✅